dateparser/search/search.py

```diff
@@ -86,10 +86,15 @@ def split_if_not_parsed(self, item, original):
                 possible_splits.extend(self.split_by(item, original, splitter))
         return possible_splits
 
-    def parse_item(self, parser, item, translated_item, parsed, need_relative_base):
+    def parse_item(self, parser, item, translated_item, parsed, need_relative_base, language):
         relative_base = None
-        item = item.replace('ngày', '')
-        item = item.replace('am', '')
+
+        if language != "en":
+            item = item.replace('ngày', '')
+            item = item.replace('am', '')
+
+
+        #CHANGE HERE
```

Comment:
I don't think this comment should be here.

Reply:
```suggestion
        #CHANGE HERE
```


Thanks, I have removed the comments